db/re/item_db_equip.yml

```
@@ -39118,7 +39118,7 @@ Body:
       bonus bAgiVit,10;
       bonus bInt,5;
       bonus bLuk,20;
-      itemskill "NPC_HELLJUDGEMENT",10;
+      skill "NPC_HELLJUDGEMENT",10;
```

if I refer to aegis item script it was correct (our `itemskill` should be the equivalent to their `Skill`)
```
item Horn_Of_Lord_Kaho
	event OnStartEquip:
		Skill	NPC_HELLJUDGEMENT 	10
		return
```
```suggestion
      itemskill "NPC_HELLJUDGEMENT",10;
```